bin/apisix

```diff
@@ -706,28 +706,48 @@ local function init_etcd(show_output)
     end
 
     local etcd_conf = yaml_conf.etcd
-    local uri = etcd_conf.host .. "/v2/keys" .. (etcd_conf.prefix or "")
 
     local timeout = etcd_conf.timeout or 3
+    local uri
+    local host_count = #(yaml_conf.etcd.host)
```

Comment:
if the `yaml_conf.etcd.host` is a string object, it will get the string length of it.

Reply:
oh sure @membphis